frontend/src/pages/edit-user-page/index.css

```
@@ -0,0 +1,31 @@
+h2 {
+    font-size: 20px;
+    display: flex;
+    justify-content: left;
+    align-items: left;
+  }
+
+  .boxContainer {
```

flush this to the left side and fix spacing
```suggestion
.boxContainer {
```

---

dont we already have another boxContainer class defined in another page? please make a global css file and call the boxContainer from there

---

update the other files using the boxContainer as well